tasks/build.js

```diff
@@ -55,9 +56,28 @@ async function api() {
     }
 }
 
+async function executeChildProcess(args) {
+    if (process.env.BUILD_CHILD) {
+        throw new Error('Infinite loop');
+    }
+    const env = {...process.env, BUILD_CHILD: '1'};
+    const child = fork(__filename, args, {env});
+    // Send SIGINTs as SIGKILLs, which are not ignored
+    process.on('SIGINT', () => {
+        child.kill('SIGKILL');
+        process.exit(130);
```

Comment:
Why 130?

Reply:
Well, for Linux, a fatal error is 128, and they add the signal code to that, which in this case is 2. At least I think that's how it goes.

In this case, it just needs to be non-zero. I can do that if you want